docs/source/basic_tutorials/request_parameters.md

```diff
@@ -0,0 +1,29 @@
+# Request Parameters for Text Generation Inference
+
+Text Generation Inference supports different parameters to control the generation, defining them in the `parameters`` attribute of the payload. 
+
+```bash
+curl https://j4xhm53fxl9ussm8.us-east-1.aws.endpoints.huggingface.cloud \
+-X POST \
+-d '{"inputs":"Once upon a time,", "parameters": {"max_new_tokens": 256}}' \
+-H "Authorization: Bearer <hf_token>" \
+-H "Content-Type: application/json"
+```
+
+
+As of today, the following parameters are supported:
```

Comment:
```suggestion
As of today, the following parameters are supported:
```
small heads up before moving on with more PRs, for explaining e.g. params we tend to rely on self generating docs (e.g. what you see in swagger)